R/commonMachineLearningClassification.R

```diff
@@ -564,6 +586,26 @@
     fit <- e1071::naiveBayes(formula, data = dataset, laplace = options[["smoothingParameter"]])
     predictions <- as.factor(max.col(predict(fit, newdata = grid, type = "raw")))
     levels(predictions) <- unique(dataset[, options[["target"]]])
+  } else if (type == "logistic") {
+    if (classificationResult[["family"]] == "binomial") {
+      fit <- glm(formula, data = dataset, family = stats::binomial(link = "logit"))
+      predictions <- as.factor(round(predict(fit, grid, type = "response"), 0))
+      levels(predictions) <- unique(dataset[, options[["target"]]])
+    } else {
+      fit <- VGAM::vglm(formula, data = dataset, family = VGAM::multinomial())
+      logodds <- predict(fit, newdata = grid)
```

Comment:
```suggestion
      logodds <- stats::predict(fit, newdata = grid)
```
we should really turn on the R CMD check for things like this, would be way easier to catch them all.

Reply:
There are many more calls to predict that do not have this in the module. Suggestion: do a cleaning run for this sometime together with implementing this R CMD check.